### DISCUSSION TITLE
Update deprecating web3auth v7 to v8 in @alchemy/aa-signers

### DISCUSSION THREAD

**Discussion by krisliew** [alchemyplatform/aa-sdk#898] [Issues]
Greetings Alchemy Team,

Any idea if there will be an update for the web3auth package that were used in the `@alchemy/aa-signers` package from v7 (deprecating) to v8?

https://github.com/alchemyplatform/aa-sdk/blob/d067879431403ce9acdc36841fd66deb2bfc7223/packages/signers/package.json#L110-L111

**Comment by avarobinson** [comment]
Hi @krisliew - we are working on upgrading this. Thanks for flagging!
